cogs/tagging.py

```diff
@@ -48,24 +48,25 @@ async def tag_add(self, ctx, name, *, content: str):
 
         db.taggingEntry.insert(ctx.message.channel.id, name, content, ctx.message.created_at)
         await ctx.send(f'Tag "{name}" created.')
-    
-    @commands.command()
+
+    @tag.command()
     @commands.has_role("Library Developer")
```

Comment:
Actually, why do we need to check if the sender is a lib dev here?

Reply:
Editing & creating tags are only for library developers. Fetching tags is for anyone.